module-system/module-implementations/sov-evm/src/signer/mod.rs

```diff
@@ -57,6 +58,25 @@ impl DevSigner {
         ))
     }
 
+    pub fn sign_ethers_transaction(
```

Comment:
Could be replaced with: https://github.com/paradigmxyz/reth/blob/main/crates/rpc/rpc/src/eth/signer.rs#L74